botocore/data/autoscaling/2011-01-01/waiters-2.json

```diff
@@ -0,0 +1,62 @@
+{
+    "version": 2,
+    "waiters": {
+        "GroupExists": {
+            "acceptors": [
+                {
+                    "argument": "length(AutoScalingGroups) > `0`",
```

Comment:
Yes.  This is a "wait until at least one group exists out of the N you might have specified."